Makefile

```diff
@@ -115,6 +115,12 @@ ML_DESCRIPTION := "Coq v8.14"
 OTHERFLAGS += -w "-deprecated-appcontext -notation-overridden"
 ML4_OR_MLG := mlg
 else
+ifneq (,$(filter 8.15%,$(COQ_VERSION)))
+EXPECTED_EXT:=.v815
+ML_DESCRIPTION := "Coq v8.15"
+OTHERFLAGS += -w "-deprecated-appcontext -notation-overridden"
```

Comment:
appcontext hasn't existed for a while

Reply:
I cargo-culted the code for the 8.14 version, don't slap my hands.